CorinneTest/CorinneTest.sln

```diff
@@ -0,0 +1,17 @@
+

```

Comment:
we probably didnt want this committed but ill just take it out after the PR is merged

Reply:
certain files are in the ".gitignore" which means they wont be tracked for modifications. this is useful for generated stuff like /bin/ or /obj/ . In java parlance youd want to save all of your .Java classes, but not your .Class compiled stuff since thats not useful to anyone